### ISSUE TITLE
Show node, and serverID

### ISSUE THREAD

**Issue by Hadimhz** [pterodactyl/panel#2852]
**Is your feature request related to a problem? Please describe.**
Nope

**Describe the solution you'd like**
I'd like to see the server identifier and name are visible from the server page.
maybe in the info box??
![image](https://user-images.githubusercontent.com/40409775/101802007-a09f1980-3b17-11eb-9ac0-e01b479d14dc.png)

**Comment by DaneEveritt** [comment]
Probably makes more sense under Settings, no? It isn't exactly needed for much other than API calls, and it is already present in the page URL.

**Comment by ElijahRus250** [comment]
It can be used for support with hosts, however it would not be very hard to implement with some code editing

**Comment by tinyoverflow** [comment]
> Probably makes more sense under Settings, no? It isn't exactly needed for much other than API calls, and it is already present in the page URL.

To make it short, but hard: Customers are stupid. I've learned this the hard way when I've worked in customer support for a hosting company for a year. Yes, some of them do have a technical understanding, but most of them aren't even able to read a documentation you've send them the link via email.

I'm definitely with you: It might be useful for the customer to see in some situations, but I also prefer this on the settings page. @ElijahRus250 is right, it probably would be an easy implementation.
